### ISSUE TITLE
It would be nice if it was possible to change the white color during the screenshot

### ISSUE THREAD

**Issue by iambeingtracked** [flameshot-org/flameshot#2841]
### Feature Description

How it currently looks: https://0x0.st/ojE-.png
How I would like it to look (the color used here is #d3c6aa, but it should be possible to customize that color, for example in flameshot.ini):https://0x0.st/ojEN.png
Sorry for my poor coloring, I think you'll understand what I meant

**Comment by mmahmoudian** [comment]
It is already possible. Download the latest version (12.1.0), then `flameshot config' and you can change it. It is also possible to add the color to your config. For example, add `uiColor=#ff7300` to the `[General]` section of your config file.

Does this address the issue?

**Comment by iambeingtracked** [comment]
No, `uiColor` changes the color that is purple by default, I want to change the white color that is used mostly for text. See the screenshots I linked to better understand what I meant

**Comment by mmahmoudian** [comment]
That color is selected automatically based on the color you select as your uiColor. For example in the following Gif I'm changing the darkness of the color and you can see that the cirle in the middle of the uiColor is switching between dark and light based on the how dark or light the color I select is:

![Peek 2022-07-28 17-06](https://user-images.githubusercontent.com/390889/181525724-f2c0ebfe-6bbe-4831-8580-13d7ba13dced.gif)

**Comment by iambeingtracked** [comment]
But I want to choose that color myself, and afaik it is not possible.

**Comment by mmahmoudian** [comment]
> afaik it is not possible

Yes, it is not possible in the current implementation. It is chosen algorithmically based on the color you have chosen. May I ask what the use case is? is it just for making it look more beautiful or there are some colors that the contrast is not high enough and the button signs are not visible?

**Comment by iambeingtracked** [comment]
I just want it to look as I want, everything is fine but I'd like to choose colors by myself.
